src/argilla/client/feedback/dataset.py

```diff
@@ -234,10 +235,12 @@ def __init__(
         any_required = False
         unique_names = set()
         for question in questions:
-            if not isinstance(question, (TextQuestion, RatingQuestion, LabelQuestion, MultiLabelQuestion)):
+            if not isinstance(
+                question, (TextQuestion, RatingQuestion, LabelQuestion, MultiLabelQuestion, RankingQuestion)
```

Comment:
Maybe we can externalize this tuple as `SUPPORTED_QUESTION_TYPES` constant and reuse in different places

Reply:
We already have `AllowedQuestionTypes`, but that's a `typing.Union` so we cannot re-use it, but yes, we can create a list of types and re-use it here